docs/doc/50-manage/01-query/20-query-metrics.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Databend Query Metrics
+sidebar_label: Databend Query Metrics
+description:
+  Databend Query Metrics
+---
+
+A `databend-query` server records metrics in table [system.metrics](../../30-reference/30-sql/70-system-tables/system-metrics.md).
+
+The [metric_api_address](../../50-manage/01-query/10-query-config.md) to listen on that can be scraped by Prometheus and will return a [Prometheus](http://prometheus.io/docs/instrumenting/exposition_formats/) format of metrics.
+
+```sql
+mysql> DESC system.metrics;
++--------+---------+------+---------+-------+
+| Field  | Type    | Null | Default | Extra |
++--------+---------+------+---------+-------+
+| metric | VARCHAR | NO   |         |       |
+| kind   | VARCHAR | NO   |         |       |
+| labels | VARCHAR | NO   |         |       |
+| value  | VARCHAR | NO   |         |       |
++--------+---------+------+---------+-------+
+
+```
+
+Metrics has four attributes: `metric`, `kind`, `labels`, `value`.
+
+`metric` indicate the name of the metric contained in the databend-query.
+
+`kind` records the kind of metric. Now contains [`counter`](https://prometheus.io/docs/concepts/metric_types/#counter), [`gauge`](https://prometheus.io/docs/concepts/metric_types/#gauge), [`summary`](https://prometheus.io/docs/concepts/metric_types/#summary) and [`untyped`](https://prometheus.io/docs/concepts/metric_types/#metric-types).
+
+`labels` records the label of metric.
+
+`value` records the value of metric. 
+
+## Metrics
+
+```sql
+mysql> SELECT * FROM system.metrics;
```

Comment:
This is duplicated in `show-metrics.md`.

I think we just need to put a table which describes the metric.

Reply:
I think the `show-metrics.md` just used to display the result of the query `show metric`. And tell us how to use `show metrics`.

But the `system-metrics.md ` should not only display a result.

Reply:
Maybe it's better to add a link in `system-metrics.md` or `show-metrics.md`. So that user can jump to this file.